CONTRIBUTING.md

```diff
@@ -105,7 +105,7 @@ Instructions for a minimal build environment without conda are included below.
 # create the conda environment (assuming in base `cudf` directory)
 # note: RAPIDS currently doesn't support `channel_priority: strict`;
 # use `channel_priority: flexible` instead
-conda env create --name cudf_dev --file conda/environments/all_cuda-118_arch-x86_64.yaml
+conda env create --solver=libmamba --name cudf_dev --file conda/environments/all_cuda-120_arch-x86_64.yaml
```

Comment:
Since `libmamba` is the default solver for more recent versions of `conda`, IMO it's not strictly necessary to change the solver here

Reply:
Good point. I saw it mentioned in https://docs.rapids.ai/install#selector and thought I would add it here. I'm happy to take out if preferred.

Reply:
This file was changed to a CUDA 12.2 environment. I agree with @mroeschke about not needing the solver here.

I'll update this and merge.
```suggestion
conda env create --name cudf_dev --file conda/environments/all_cuda-122_arch-x86_64.yaml
```